devel/spicy/tutorial.rst

```diff
@@ -82,34 +82,43 @@ protocol analyzer. We follow the same scheme here and put the
 following into ``tftp.evt``, the analyzer definition file:
 
 .. literalinclude:: autogen/tftp.evt
-    :lines: 5-7
+    :lines: 5-6
     :language: spicy-evt
 
 The first line provides our analyzer with a Zeek-side name
 (``spicy::TFTP``) and also tells Zeek that we are adding an
 application analyzer on top of UDP (``over UDP``). ``TFTP::Packet``
 provides the top-level entry point for parsing both sides of a TFTP
-connection. Furthermore, we want Zeek to automatically activate our
-analyzer for all sessions on UDP port 69 (i.e., TFTP's well known
-port). See :ref:`spicy_evt_analyzer_setup` for more details on defining
+connection. See :ref:`spicy_evt_analyzer_setup` for more details on defining
 such a ``protocol analyzer`` section.
 
+Furthermore, we want Zeek to automatically activate our analyzer for all
+sessions on UDP port 69 (i.e., TFTP's well known port) using the following
+script snippet:
+
+.. literalinclude:: examples/tftp-register-analyzer.zeek
+    :language: zeek
+
+This snippet can generally be assumed to be part of the analyzer's scripts.
+We'll reference it as ``tftp-register-analyzer.zeek`` below.
+
+
 With this in place, we can already employ the analyzer inside Zeek. It
 will not generate any events yet, but we can at least see the output of
 the ``on %done { print self; }`` hook that still remains part of the
 grammar from earlier:
 
 .. code::
 
-    # zeek -r tftp_rrq.pcap tftp.hlto Spicy::enable_print=T
+    # zeek -r tftp_rrq.pcap tftp.hlto tftp-register-analyzer.zeek Spicy::enable_print=T
```

Comment:
> (I have to say: I liked being able to not need a Zeek script for this example ...)

I also thought that `port` / `ports` was kind of nice for the whole tutorial. Oh well :-(

Had a `zeek ... -e 'Analyzer::register_for_port(...)` in mind, but probably more confusing in the end than having it in a scirpt.

I'm tempted to name the script `tftp.zeek` like all the other examples, so it stands out a bit less (and the other examples build upon each other, so that makes some sense).

Reply:
> I'm tempted to name the script tftp.zeek like all the other examples, 

Good though, agree.